pym/bob/audit.py

```diff
@@ -84,6 +84,7 @@ class Artifact:
             'date'     : str
         },
         "env" : str,
+        "metaEnv" : { schema.Optional(str) : str },
```

Comment:
We have to make the metaEnv key optional. Otherwise existing audit trails in old workspaces will fail builds.